ampersand-select-view.js

```diff
@@ -133,7 +135,7 @@ SelectView.prototype.renderOptions = function () {
 
     this.options.forEach(function (option) {
         this.select.appendChild(
-            createOption(this.getOptionValue(option), this.getOptionText(option))
+            createOption(this.getOptionValue(option), this.getOptionText(option), this.getOptionDisabled(option))
```

Comment:
@kamilogorek The createOption call is an argument for the appendChild method so I don't think there should be a semicolon in this instance?